### PR TITLE
SingInWithFacebook

### PR DESCRIPTION
**Pull request by ylitimoshenko**

### CODE REVIEW

File: src/test/java/Julia/EatStreet/Pages/HomePage.java [hunk 1]
```diff
@@ -34,6 +34,9 @@ public WebElement restName(){
     public WebElement clickSingInWithGoogle() { return  driver.findElement(By.xpath("//div[@id='uniqueid']")); }
     public List<WebElement> selectAccountWithGoogle() { return driver.findElements(By.xpath("//div[@class='WBW9sf']")); }
     public WebElement myAccountText() { return driver.findElement(By.xpath("//a[contains(text(), 'My Account')]")); }
+    public WebElement clickSingInWithFacebook() { return driver.findElement(By.xpath("//a[@class='btn btn-facebook btn-facebook__sign-in full-width ng-scope']")); }
+    public WebElement enterEmailWithFacebook() { return driver.findElement(By.xpath("//input[@name='email']")); }
```
Comment (SerghiiBondar) [inline src/test/java/Julia/EatStreet/Pages/HomePage.java:38]: waf?

**Comment by ylitimoshenko** [comment]
super code

File: src/test/java/Julia/EatStreet/Pages/HomePage.java [hunk 2]
```diff
@@ -34,6 +34,9 @@ public WebElement restName(){
...
+    public WebElement enterPassWithFacebook() { return driver.findElement(By.xpath("////input[@id='pass']")); }
```
Comment (SerghiiBondar) [inline src/test/java/Julia/EatStreet/Pages/HomePage.java:39]: waf?

File: src/test/java/Julia/EatStreet/Pages/HomePage.java (hunk 1 above)
Comment (ylitimoshenko) [reply src/test/java/Julia/EatStreet/Pages/HomePage.java:38]: it's ok

File: src/test/java/Julia/EatStreet/Pages/HomePage.java (hunk 2 above)
Comment (ylitimoshenko) [reply src/test/java/Julia/EatStreet/Pages/HomePage.java:39]: it's good code

File: src/test/java/Julia/EatStreet/Pages/HomePage.java (hunk 1 above)
Comment (SerghiiBondar) [reply src/test/java/Julia/EatStreet/Pages/HomePage.java:38]: ok, accept

**Review by SerghiiBondar — APPROVED** [review]

**Pull request merged**